### ISSUE TITLE
Call timing out at the 25 min mark.

### ISSUE THREAD

**Issue by LionSalesFunnels** [0perationPrivacy/VoIP#127]
- **Platform:**  [ Mobile]
- **Browser:** [ Brave]
- **App Version:** [e.g v0.89]
- **Provider:** [Twilio]
- ### Describe the bug
Placing an outbound call in the same country and area code. Experiencing an in browser reload at the 25 min mark that drops the call. The browser seems to be crashing. The screen is kept on. The battery optimization is turned off for Brave. The app is kept alive but crashes.

**To Reproduce**
1. Go to 'open brave'
2. dial a call
3. keep the call active to 25 minutes
4. See browser reloading

**Expected behavior**
Being able to maintain a call up to 75 minutes + without dropout should be expected.

**Comment by LionSalesFunnels** [comment]
Here is the log. Is everything normal?
![TwilioApp_DropOut_55](https://user-images.githubusercontent.com/56514208/161393553-9daa85e8-9193-4f2b-af8b-ab274b2530dc.png)

**Comment by LionSalesFunnels** [comment]
No errors on the Twilio log.

**Comment by 0perationPrivacy** [comment]
Check the Heroku logs to see if you get any error there. I have not tested it over 25mins!
Will check if something refreshing the page may be causing a call drop.

**Comment by LionSalesFunnels** [comment]
Sorry for asking but what is the folder path of the log on the server build?

Thank you

**Comment by 0perationPrivacy** [comment]
> Sorry for asking but what is the folder path of the log on the server build? Thank you

In your Heroku dashboard, on the upper right there's `More` and under that click `View Logs`.

**Comment by LionSalesFunnels** [comment]
I don't believe our setup is using an Heroku dashboard. Can't seem to find the aforementioned menu options.

**Comment by VoiceIPRules** [comment]
I'm confirming that we don't use Heroku. Where can we setup our own logs?  

We are running Docker on a VPN.
Please help. Any clues?

**Comment by VoiceIPRules** [comment]
Still the same problem call dropping / browser reloading after 25 minutes after the latest update.

Any clues?. Any preferences for browser?

**Comment by 0perationPrivacy** [comment]
> Still the same problem call dropping / browser reloading after 25 minutes after the latest update.
> 
> Any clues?. Any preferences for browser?

I checked the code and had a programmer check it as well, we couldn't find anything that refreshes the session or page after 25 mins. Could be be something related to your particular environment or browser. I have not tested this on Docker.
